### PR TITLE
chore: don't not upload/compress test canisters

### PR DESCRIPTION
**Pull request by rumenov**
removed canisters are 
```
     "//rs/rust_canisters/pmap:pmap_canister",
     "//rs/rust_canisters/proxy_canister:proxy_canister",
     "//rs/sns/integration_tests:sns-governance-mem-test-canister",
```

### CODE REVIEW

**Comment by rumenov** [comment]
Please review with care

**Review by mbjorkqvist — APPROVED** [review]

File: publish/canisters/BUILD.bazel [hunk 1]
```diff
@@ -1,32 +1,25 @@
...
-# 2. Canisters required for dfx - https://github.com/dfinity/dfx-extensions/tree/main/extensions-utils/src/dependencies/download_wasms
-# 3. Mainnet canisters - https://github.com/dfinity/ic/blob/master/mainnet-canisters.bzl
+# 2. Mainnet canisters - https://github.com/dfinity/ic/blob/master/mainnet-canisters.bzl
+# 3. Canisters required for dfx - https://github.com/dfinity/dfx-extensions/tree/main/extensions-utils/src/dependencies/download_wasms
 CANISTERS = {
     "canister-creator-canister.wasm.gz": "//rs/rust_canisters/canister_creator:canister_creator_canister",
     "cycles-minting-canister.wasm.gz": "//rs/nns/cmc:cycles-minting-canister",
     "genesis-token-canister.wasm.gz": "//rs/nns/gtc:genesis-token-canister",
     "governance-canister.wasm.gz": "//rs/nns/governance:governance-canister",
-    "governance-canister_test.wasm.gz": "//rs/nns/governance:governance-canister-test",
```
Comment (max-dfinity) [inline publish/canisters/BUILD.bazel:18]: I think this may be used externally in testing, along with the test version of the sns-governance-canister

Comment (max-dfinity) [reply publish/canisters/BUILD.bazel:18]: Yes, it is. `dfx nns install` uses the test governance canister (thanks @dskloetd for confirming)

Comment (rumenov) [reply publish/canisters/BUILD.bazel:18]: Who may know if it is used or not ? Also what type of testing exactly ?

Comment (rumenov) [reply publish/canisters/BUILD.bazel:18]: Thanks @max-dfinity

**Review by max-dfinity — APPROVED** [review]

**Pull request merged**